### ISSUE TITLE
Consider documenting that `rvk` denotes expiry for Coze keys.  

### ISSUE THREAD

**Issue by zamicol** [Cyphrme/Coze#21]
The word revoke may connotate that a key was never trusted/authorized.  

The specification should better explain that `rvk` is expected to be an expiry time.  

Cozies signed before the rvk time should be considered valid and action signed after should be ignored.  

As is currently noted by the specification, actionable events based on future expiration times are outside the scope of Coze, and revoke messages with future times should result in the signing key considered immediately expired.  

Thanks @qbit for bringing up this concern.

**Comment by zamicol** [comment]
Revision:

## Revoke
A Coze key may be revoked by signing a coze containing the field `rvk` with an
integer value greater than `0`.  For example, the integer value `1` is suitable
to denote revocation.  The current Unix timestamp is the suggested value for
`rvk`.


### Example Self Revoke
```JSON
{
 "pay": {
  "alg": "ES256",
  "iat": 1623132000,
  "msg": "Posted my private key online",
  "rvk": 1623132000,
  "tmb": "cLj8vsYtMBwYkzoFVZHBZo6SNL8wSdCIjCKAwXNuhOk",
  "typ": "cyphr.me/key/revoke"
 },
 "sig": "KVjPjMVHoL828WyAH5biqIOt-IOaQ5EBtN_7eQifP2w3agUHu6KfqO40_oqQ5GE_BShgXvhbK0O6Z2h5YPNAcw"
}
```

- `rvk` - Unix timestamp of key expiry.

Coze explicitly defines a self-revoke method so that third parties may revoke
leaked keys. Systems storing Coze keys should provide an interface permitting a
given Coze key to be marked as expired by receiving a self-revoke message.
Self-revokes with future times must immediately be considered as expired.

`rvk` and `iat` must be a positive integer less than 2^53 – 1
(9,007,199,254,740,991), which is the integer precision limit specified by
IEEE754 minus one. Revoke checks must error if `rvk` is not an integer or larger
than 2^53 - 1.

Key expiration policies, key rotation, backdating, and alternative revocation
methods are outside the scope of Coze.

**Comment by zamicol** [comment]
Fixed by 5af99fd
